### DISCUSSION TITLE
符华

### DISCUSSION THREAD

**Discussion by 566001** [share121/inter-knot#1862] [其他]
(![image](https://github.com/user-attachments/assets/1c180bdf-3911-4af4-b108-7bded6a24f67))
崩坏3中登场的首位中国角色女武神。天命所属A级女武神，琪亚娜等人在圣芙蕾雅学园学习时所属班级的班长。不论是理论学习还是战斗实践都十分优秀，是一个典型的优等生。使用中华武术格斗。

红框眼镜，奥特·赛文变身用同款(误)。作为通讯设备可以与奥托秘密联络，同时也是接驳装甲的接口。

动作十分敏捷，能够通过风骚的走位躲避敌人的各种攻击。防御能力不重要，反正打不中。

擅长使用中华武术进行近身格斗，并以一拳快速结束战斗。（一拳超人）

在逆熵入侵篇里面因为没有穿大腿袜所以可能无法成为抑制崩坏的核心力量。

在月影篇里符华也穿上了大腿袜，由此获得了主角光环。

然而对战瓦二特的时候却不知为何又把大腿袜脱了下来，为此烧鸡娘还专门地拿出一期崩坏教室的篇幅一本正经地分析（胡扯）了原因

持有第十神之键数把，剧情已出现的为：太虚之握、轩辕剑共七把（分别是给程立雪的若水，给7徒的6把，可能还有更多，但剧情只出现这些）和第八神之键：羽渡尘。

一生收了八个徒弟：“轻尘剑”林朝雨，“无双仙子”苏湄，（双姐妹：江婉兮/江婉如），“无上自在”程凌霜，“逐驹剑”马非马，秦素衣，“寒霜剑客”程立雪。

三厨神之一（另外两位是芽衣和丽塔），做得一手好中国菜。其实符华原本做菜难吃，现纪元文明伊始跟丹朱苍玄学了一手做菜本领（来自女武神的餐桌漫画剧情），后历经千年磨炼终于练出一番好手艺。

**Comment by gaojiaobei** [comment]
班长！豪堪🥰
